src/aot/aot_main.cpp

```diff
@@ -63,7 +62,8 @@ std::unique_ptr<Output> prepare_output(const std::string& output_file,
 
 int main(int argc, char* argv[])
 {
-  std::string output_file, output_format;
```

Comment:
why is this bad?

Reply:
I have no preference. Just what clang-tidy seems to prefer.

This is the check: https://clang.llvm.org/extra/clang-tidy/checks/readability/isolate-declaration.html

Reply:
Eh, we work in the kernel code base where this is not only allowed but encouraged. I'm sure there are reasons to prefer clang-tidy's way but I feel like it might be too prescriptive - like would we want to block a PR because of this?

Reply:
Good point. I will revert the isolate-declaration check